.github/workflows/main.yml

```diff
@@ -135,10 +135,10 @@ jobs:
         run: make tidy-module-check
 
       - name: install protolint
-        run: ./scripts/install_protolint.sh
+        run: make protolint-install
```

Comment:
The two commits can be squashed so we don't need to update the same file in both commits.